### PR TITLE
Localize menu userscript details

### PR DESCRIPTION
**Pull request by Eselce**
Localizes newly added userscript details in `Greasemonkey` menu

### CODE REVIEW

File: _locales/en/messages.json [hunk 1]
```diff
@@ -97,6 +97,15 @@
   "matches": {
     "message": "Matches:"
   },
+  "menu_description": {
```
Comment (arantius) [inline _locales/en/messages.json]: We don't anywhere else, and I don't want to get into the business of, trying to "categorize" the strings.

If the string is "Description", its name should be "description", and we should be able to use it anywhere we want to use that string.

Comment (Eselce) [reply _locales/en/messages.json]: Okay!

Comment (Eselce) [reply _locales/en/messages.json]: How will we differ labels and objects, i.e. for example ```Version:``` vs. ```version```?
I'd know the answer for ```Version $version$``` (that would be ```version_VERSION```)...

File: src/browser/monkey-menu.html [hunk 2]
```diff
@@ -118,9 +118,9 @@
 
     <hr>
 
-    <p rv-if="activeScript.description">{'description'|i18n}
+    <p rv-if="activeScript.description"><strong>{'description'|i18n}</strong>
```
Comment (arantius) [inline src/browser/monkey-menu.html]: I'm not sure how you picked this HTML style, but I'd greatly prefer:

<p rv-if="activeScript.description">
  <strong>{'description'|i18n}</strong> {activeScript.description}
</p>

(Maybe a linebreak before `{` if that's >80 chars.

Comment (arantius) [reply src/browser/monkey-menu.html]: No, it was strong (copy/paste and edit of what was there), either is fine.  Bah markdown for eating (some) HTML tags.

Comment (Eselce) [reply src/browser/monkey-menu.html]: Your decision: &lt;b&gt; or &lt;strong&gt;, both are okay. I've started with &lt;u&gt; (which was really okay, but one should probably avoid underline here). The install dialog used &lt;strong&gt;. Current implementation is &lt;b&gt;.

**Pull request merged**